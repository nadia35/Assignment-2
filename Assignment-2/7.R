cloneAnimals <- function(my_animals, times_cloned){
  
  times_cloned <- times_cloned + 1
  my_animals <- sort(rep(my_animals, times_cloned))
  
  return(my_animals)
}
