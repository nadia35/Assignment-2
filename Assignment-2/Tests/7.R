source("7.R")

mammals = c("rhino", "elephant", "lion", "tiger", "hippo", "cheetah", "panda")
birds = c("parrot", "flamingo", "ostrich", "owl", "emu", "peacock")
fish = c("shark", "piranha")
amphibians = c("frog", "toad", "newt", "salamander", "mantella")

print(cloneAnimals(mammals, 1)) 
print(cloneAnimals(birds, 2)) 
print(cloneAnimals(fish, 5)) 
print(cloneAnimals(amphibians, 0))