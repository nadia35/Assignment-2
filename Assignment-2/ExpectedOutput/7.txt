 [1] "cheetah"  "cheetah"  "elephant" "elephant" "hippo"    "hippo"   
 [7] "lion"     "lion"     "panda"    "panda"    "rhino"    "rhino"   
[13] "tiger"    "tiger"   
 [1] "emu"      "emu"      "emu"      "flamingo" "flamingo" "flamingo"
 [7] "ostrich"  "ostrich"  "ostrich"  "owl"      "owl"      "owl"     
[13] "parrot"   "parrot"   "parrot"   "peacock"  "peacock"  "peacock" 
 [1] "piranha" "piranha" "piranha" "piranha" "piranha" "piranha" "shark"  
 [8] "shark"   "shark"   "shark"   "shark"   "shark"  
[1] "frog"       "mantella"   "newt"       "salamander" "toad"      
